docs/cocos-sdk-quickstart.md

```diff
@@ -38,7 +38,7 @@ Loom Cocos SDK structure diagram
 
 `Loom Cocos SDK` is based on [Loom-JS](https://github.com/loomnetwork/loom-js/) , and porting it to `Cocos Creator` .
 
-### Generate LoomJS SDK
+### Generate Cocos SDK
```

Comment:
I think this section is about generating Cocos SDK not LoomJS SDK, so I changed title like above.